<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Dataset Validator</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Image Dataset Validator</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtLocation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScan">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>548</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstLog">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>701</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Location: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>652</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
